contrib/goattracker/files/goattracker.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=GoatTracker
+Comment=C64 music editor
+Exec=goattracker %F
+Icon=goattracker.png
```

Comment:
and following the below, you don't need the .png extension here, just the bare name (extension is auto picked)